app/models/trip.rb

```diff
@@ -96,37 +96,65 @@ def self.count_of_fewest_trips
   end
 
   def self.number_of_starting_rides_at_station(station_id)
-    stations = Trip.group(:start_station_id).count(:id)
-    stations[station_id]
+    if Trip.where(start_station_id: station_id) == []
```

Comment:
I think this is good/best practice for us on things like this in the future, even if it wasn't really our problem, after all. Way to go!